apps/sel4test-tests/src/tests/schedcontext.c

```diff
@@ -12,6 +12,10 @@
 
 #include "../helpers.h"
 
+#ifndef MIN_BUDGET_US
+#define MIN_BUDGET_US (2 * 10)
+#endif
```

Comment:
So, I merged this a bit too early -- this breaks for TK1 where KERNEL_WCET is set to 100 instead of 10

Reply:
Best fix would be to export MIN_BUDGET_US in libsel4.

Reply:
Alternatively, stop the tests without error if the api_sched_ctrl_configure() call fails.

Reply:
I couldn't figure out how to export MIN_BUDGET_US from seL4, so instead changed the hardcoded MIN_BUDGET_US define to 100 for TK1. Not sure if CONFIG_PLAT_TK1 is set, but that's what others are using. See commit b2c24b0.

Reply:
Thanks, I added a comment and made that into a PR. Exporting MIN_BUDGET_US is indeed a bit annoying because drags a few other things with it.